### PR TITLE
[MWPW-160730] - Project Red logo update for adobecom and bacom

### PR DESCRIPTION
**Pull request by Deva309**
Add changes for logo update on adobecom and bacom pages.

### Below are the assets for new logos used with demo Url's:
- [https://main--federal--adobecom.hlx.page/federal/assets/svgs/adobe-wordmark-red-resize.svg](https://main--federal--adobecom.hlx.page/federal/assets/svgs/adobe-wordmark-red-resize.svg) (adobecom)
- [https://main--federal--adobecom.hlx.page/federal/assets/svgs/lockup-adobe-experience-cloud-one-line-red-black-resize.svg](https://main--federal--adobecom.hlx.page/federal/assets/svgs/lockup-adobe-experience-cloud-one-line-red-black-resize.svg) (bacom)

### Authoring changes needed:
**Case 1**: If replacing same old logo with new logo then we need to just add identifier as `Brand Image Only` for `Gnav Brand` block. Refer below Image:
Screenshot 2024-10-30 at 3 30 03 PM

**Case 2**: If using new logo with different name or path then we need to update identifier as `Brand Image Only` for `Gnav brand` block and update the new logo URL accordingly. Refer below image:
Screenshot 2024-10-30 at 3 29 04 PM

**Wiki for details**: https://wiki.corp.adobe.com/display/WEBT/%5BProject+Red%5D+-+Adobe+Logo+changes

Resolves: [MWPW-160730](https://jira.corp.adobe.com/browse/MWPW-160730)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/?martech=off
- After: https://mwpw-160730--milo--deva309.hlx.page/?martech=off

### CODE REVIEW

**Comment by salonijain3** [comment]
this is a high priority PR and is needed for the testing of the Logo changes.

File: libs/blocks/global-navigation/global-navigation.js [hunk 1]
```diff
@@ -760,8 +760,9 @@ class Gnav {
     if (!link) return '';
 
     // Check which elements should be rendered
+    const isBrandImage = rawBlock.matches('.brand-image');
```
Comment (bandana147) [inline libs/blocks/global-navigation/global-navigation.js:763]: Should we keep the class name while authoring to brand-image-only?

Comment (Deva309) [reply libs/blocks/global-navigation/global-navigation.js]: Done

**Comment by Deva309** [comment]
Please use below Url's for QA testing on different logos:
https://mwpw-160730--milo--deva309.hlx.page/drafts/devashish/default/no-logo-page (Default Logo)
https://mwpw-160730--milo--deva309.hlx.page/drafts/devashish/default/adobecom-page (adobecom)
https://mwpw-160730--milo--deva309.hlx.page/drafts/devashish/default/bacom-page (bacom)

**Review by bandana147 — APPROVED** [review]

**Review by nishantka — APPROVED** [review]

**Comment by prativas22** [comment]
Adobe logo was verified across clouds using below URL in branch, and observed that, with the exception of DC, the logo image appears as expected in other clouds such as CC, Homepage, and BACOM.


**DC Cloud was impacted because some CSS overrode the global-navigation CSS :**
https://main--dc--adobecom.hlx.page/drafts/prativa/adobelogotestpage?milolibs=mwpw-160730--milo--deva309
image

Below flows are as expected 

Default Logo check:
https://main--bacom--adobecom.hlx.page/drafts/prativa/adobelogo/nologotestpage?milolibs=mwpw-160730--milo--deva309
{5EA91DFB-8A9A-4BC9-8EBD-A4B8F0600754}

{0D198DAD-955C-4F96-9897-908D8FD28848}

MILO:
https://mwpw-160730--milo--deva309.hlx.page/drafts/prativa/logo/adobelogotestpage
{9EA77FED-168A-45B9-8F2D-08889E60047A}

CC:
https://main--cc--adobecom.hlx.page/drafts/prativa/adobelogo/bacomlogotestpage?milolibs=mwpw-160730--milo--deva309
{9F79492C-A156-4C6F-81B3-6022D78C9418}
FAV icon check 
{23BC57DD-3CF0-4F26-8B11-9B07DBE36BDC}

BACOM:
https://main--bacom--adobecom.hlx.page/drafts/prativa/adobelogo/bacomlogotestpage?milolibs=mwpw-160730--milo--deva309
{867F876A-6B78-4F3C-9630-FB0785B25CD4}

Homepage:
https://main--homepage--adobecom.hlx.page/drafts/prativa/logo/adobelogotestpage?milolibs=mwpw-160730--milo--deva309
{C3C2F777-7ACA-4F19-9671-262318761C77}


CC: @salonijain3 @Deva309 @bandana147 @sigadamvenkata @spadmasa

**Comment by prativas22** [comment]
The logo for DC pages is not showing up correctly because of CSS overwrite of the global navigation CSS file, which is a blocker for the new logo from being rolled out on A.com pages.
To track the issue, a JIRA ticket [MWPW-161287](https://jira.corp.adobe.com/browse/MWPW-161287) has been created in DC. We are awaiting the DC build to proceed to STAGE.

CC: @salonijain3 @Deva309 @sigadamvenkata @spadmasa

**Comment by Deva309** [comment]
Here is the demo url's for standalone GNAV dark mode icon:
- https://adobecom.github.io/nav-consumer/navigation.html?env=stage&navbranch=mwpw-160730&authoringpath=/federal/dev/devashish&theme=dark
- https://adobecom.github.io/nav-consumer/navigation.html?env=stage&navbranch=mwpw-160730&authoringpath=/federal/dev/devashish

**Comment by spadmasa** [comment]
Validation done in below clouds 
https://main--bacom--adobecom.hlx.page/?milolibs=mwpw-160730--milo--deva309
{AE08F9A7-B37B-445A-88AC-1B65CCEB9E6A}
regression in cc , dc , bacom blog
https://main--cc--adobecom.hlx.page/creativecloud?milolibs=mwpw-160730--milo--deva309
https://main--dc--adobecom.hlx.page/acrobat/online/ai-chat-pdf?milolibs=mwpw-160730--milo--deva309
https://main--bacom-blog--adobecom.hlx.page/?milolibs=mwpw-160730--milo--deva309


Validation in cc :
https://main--cc--adobecom.hlx.page/drafts/souj/brandlogo/testlogo?milolibs=mwpw-160730--milo--deva309
https://mwpw-161287--dc--adobecom.hlx.page/drafts/prativa/adobelogotestpage?milolibs=mwpw-160730--milo--deva309
{6D119FEE-6EB6-48A2-9157-BEDF2424B78D}

https://main--homepage--adobecom.hlx.page/drafts/soujanya/doc1home?milolibs=mwpw-160730--milo--deva309
https://main--bacom--adobecom.hlx.page/products/real-time-customer-data-platform/rtcdp?milolibs=mwpw-160730--milo--deva309
https://main--bacom-blog--adobecom.hlx.page/drafts/soujanya/fgetst-website?milolibs=mwpw-160730--milo--deva309
{F01781D3-99E1-489A-8EB5-A856E6FCF460}


Standalone : 
https://adobecom.github.io/nav-consumer/navigation.html?authoringpath=/federal/test-website&customlinks=home,apps,discover,file,firefly,benefits,learn,action&navbranch=mwpw-160730&theme=dark
{802D4F1C-ACF4-48F5-9038-365F64F570C4}

https://adobecom.github.io/nav-consumer/navigation.html?authoringpath=/federal/test-website&customlinks=home,apps,discover,file,firefly,benefits,learn,action&navbranch=mwpw-160730&theme=light
{4884BC5A-EA00-4662-97A9-DCBB54F6DABF}

detailed validation results are updated in https://jira.corp.adobe.com/browse/MWPW-161270

**Pull request merged**